reana.yaml

```diff
@@ -0,0 +1,18 @@
+version: 0.9.3
```

Comment:
You can remove `version` clause that do not really serve anything and some people were confused by its meaning. (We might be removing it from everywhere later.)